src/navigation/Navigator.tsx

```diff
@@ -3,24 +3,28 @@ import React from 'react';
 import { StatusBar } from 'react-native';
 
 import { colors } from '../config';
-import { linkingConfig, navigatorConfig } from '../config/navigation';
```

Comment:
isn't the linkingConfig still needed? (Line 20)

Reply:
it is still there, from `const { navigatorConfig, linkingConfig } = navigationConfig(navigationType);`